README.md

```diff
@@ -1,44 +1,235 @@
+
+**analytics__Dashboard Target**
+
+Adding this target to the list of `<targets>` for your component allows it to be used in Tableau CRM dashboards, provided that your component is public. You can also add a `<targetConfig>` for the new target to further customize how your component appears in dashboards.
+
+**`<hasStep>` Tag**
+
+In an `analyticsDashboard` target config, you can choose to include a `<hasStep>true</hasStep>` tag to indicate that your component requires an attached step to function as expected. With this tag, the dashboard builder UI prompts you to attach an existing step or create a new step when creating an instance of your component. Components with an attached step have access to step-specific properties like `results` and `selection`. For more information on these properties, see [Tableau CRM LWC Features](#tableau-crm-lwc-features).
+
+**Measure and Dimension Attribute Data Types**
+
+Attributes specified in a LWC's `analtyicsDashboard` target config are displayed in the Tableau CRM dashboard builder UI for configuration. In addition to the common data types, this target also supports `Measure` and `Dimension` data types for components with `<hasStep>true</hasStep>`. Dashboard builders are able to choose a column of the given data type from the results of the attached step.
+
+## Tableau CRM App Templates
+
+Use the Quick Start template to practice working with Tableau CRM app templates in your scratch org. Then, create
+your own apps and templatize them using the Analytics CLI plugin and Visual Studio commands.
+
+Navigate to the sfdx-analytics directory in a terminal:
+
+1. Update the adminEmail in config/project-scratch-def.json to your email address
+
+    ```
+    sfdx force:org:open -u mydevorg
+    ```
+
+1.  Authenticate to your dev hub.
+    ```
+    sfdx force:auth:web:login -d
+    ```
+
+1.  Create a scratch org that is Analytics enabled.
+    ```
+    sfdx force:org:create -s -f config/project-scratch-def.json
+    ```
+
+1.  Push the sample Analytics template from the local workspace to the scratch org
+    ```
+    sfdx force:source:push
+    ```
+
+1. Install the Analytics SFDX plugin
+    ```
+    sfdx plugins:install @salesforce/analytics
+    ```
+
+## Tableau CRM App Template Commands
+
+1. Now you have a scratch org with an Analytics template installed.  Explore the Analytics commands by running
+    ```
+    sfdx analytics --help
+    ```
+
+1. View the options available to create an Analytics template from an app
+    ```
+    sfdx analytics:app:create --help
+    ```
+
+1. View the Analytics template:
+    ```
+    sfdx analytics:template:list
+    ```
+
+1. To create an Analytics application from the template you have two options:
+
+    1. From CLI:
+        ```
+        sfdx analytics:app:create -t <templateid>
+        ```
+
+    1. From Analytics Studio
+
+        1. Onetime: set environment variable so you always open to Analytics Page
```

Comment:
I'm not a big fan of this env var. Instead, I would say to either use the **SFDX: Open Analytics Studio** command in vscode or do `sfdx force:org:open -p analytics`.